files/image_config/hostcfgd/hostcfgd

```diff
@@ -220,6 +220,45 @@ class AaaCfg(object):
             f.write(nss_tacplus_conf)
 
 
+class k8s(object):
+    def __init__(self):
+        self.k8s_master = ""
+        self.insecure = False
+
+    # Load conf from ConfigDb
+    def load(self, data):
+        if 'SERVER' in data:
+            if 'IP' in data['SERVER']:
+                self.k8s_master = data['SERVER']['IP']
+            if 'insecure' in data['SERVER']:
+                self.insecure = data['SERVER']['insecure']
+        self.do_update()
+
+    def update(self, key, data):
+        if key == 'SERVER':
+            ip = ""
+            if 'insecure' in data:
+                self.insecure = data['insecure']
+            else:
+                self.insecure = False
+            if 'IP' in data:
+                ip = data['IP']
+            if self.k8s_master != ip:
+                self.k8s_master = ip
+                self.do_update()
+
+    def do_update(self):
+        if self.k8s_master:
+            cmd = "/usr/bin/kube_join.sh -s {} -a".format(self.k8s_master)
```

Comment:
>/usr/bin/kube_join.sh [](start = 19, length = 21)

It is python code here, so better to write kube_* in python library.

And move them into swsssdk wheel.

Reply:
Being tied to hostcfgd, I prefer to keep it in build image.